docs/lint_errors.md

```diff
@@ -6,46 +6,44 @@ This page contains detailed descriptions of the tests done by the [nf-core/tools
 nf-core pipelines should adhere to a common file structure for consistency. The lint test looks for the following required files:
 
 * `nextflow.config`
-  * The main nextflow config file
+    * The main nextflow config file
```

Comment:
You reverted all of my markdown fixes! 😱 I think the markdown linter will fail with this as it is..

Reply:
My editor ignored my settings 😱. Fixing